kyverno-policy-deployment/README.md

```diff
@@ -0,0 +1,51 @@
+# Handling Kyverno Policy Deployment
+
+Our Kyverno policies have the following properties:
+1. They are owned by different teams
+2. The need to be deployed to different environments
+3. They required different testing setups according to 1. and 2.
+
+Up until now we kept these policies in the same repository called `kyverno-policies`.
+Now that we will have to rename the repository to upgrade Kyverno is an opportune time to discuss the future structure of our Kyverno policies.
+
+## Proposals
+
+## Keep all policies in the same repository
+It's simple, everything stays the same and `kyverno-policies` is just renamed.
+### Pros
+- Not a lot of effort
+- Not much communication to make a change
+- Single source of truth for all policies
+### Cons
+- Testing setup is complex as all different environments need to be covered in one repo
+- Ownership is difficult as multiple teams own parts of the repo
+- Upgrades are cumbersome
```

Comment:
How are upgrades currently cumbersome?

Reply:
It's not clear _how_ to upgrade and where.

The policies are part of the kyverno app. So do both need new releases? It's not really clear.

It's also not clear where policies get deployed to. (MCs? WCs? All installations?)

Reply:
The policies are not part of the kyverno app. The app is shipped from https://github.com/giantswarm/kyverno-app and the policies are shipped from https://github.com/giantswarm/kyverno-policies/. The policies are deployed according to which collections they are pushed to, like other apps.

Reply:
I still think that there is also a dependency between the two repositories right now. The bigger issue is the whole collection thing though. We are pushing a bunch of charts into different collections and it's just hard to grasp where things end up (see e.g. the confusion between `shared` and `common` policies).